crates/bevy_solari/src/lib.rs

```diff
@@ -0,0 +1,42 @@
+#![expect(missing_docs, reason = "Not all docs are written yet, see #3492.")]
+
+//! Provides raytraced lighting.
```

Comment:
This should communicate the experimental nature of this crate and link to the SolariPlugin docs for more info. The SolariPlugin docs should convey the current status of the experiments, its current usefulness / current applications (or lack thereof), and plans for the future. I don't want people on main (or consuming <https://dev-docs.bevy.org>) misinterpreting the current state of things.